producers.md

```diff
@@ -35,8 +35,9 @@ Languages:
 [Prolog](#prolog),
 [Python](#python),
 [Ruby](#ruby),
+[Shell](#shell),
 and
-[Shell](#shell).
+[Fish Shell](#fish-shell)
```

Comment:
@bucaran Cool to see another shell producer! The languages list is alphabetized. When I added the languages list, I didn't intend to separate the shells from the languages by putting Shell last. It was just a coincidence because there were no other producers lower in the alphabet (i.e., if `Tcl` had an entry, it would have been at the end instead).

May I suggest that you move the "Fish Shell" line up to follow Erlang?